### PR TITLE
amneziavpn 4.1.0.1 (new cask) 

### PR DESCRIPTION
**Pull request by yslpn**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/a/amnezia-client.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+cask "amnezia-client" do
+  version "4.1.0.1"
+  sha256 "c421f28697fd9200fd9dbcdd111fa2778fe8d01b45e8ae672886399181c780cb"
+
+  url "https://github.com/amnezia-vpn/amnezia-client/releases/download/#{version}/AmneziaVPN_#{version}.dmg",
+      verified: "github.com/amnezia-vpn/amnezia-client/"
+  name "Amnezia VPN"
+  desc "Amnezia is an open-source VPN client"
```
Comment (bevanjkay) [inline Casks/a/amnezia-client.rb:8]:
```suggestion
  desc "VPN client"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]
Thanks for contributing to `homebrew-cask` @yslpn - just one small change, otherwise looks good to me.

File: Casks/a/amnezia-client.rb (hunk 1 above)
Comment (yslpn) [reply Casks/a/amnezia-client.rb]: Okay, thank you)

**Comment by bevanjkay** [comment]
@yslpn sorry for an additional change, but the token and file name for the cask should be `azeliavpn`

**Comment by yslpn** [comment]
> @yslpn sorry for an additional change, but the token and file name for the cask should be `amneziavpn `

No problem) I'll rename

**Review by bevanjkay — APPROVED** [review]
Thank you @yslpn

**Pull request merged**